COPYING

```diff
@@ -4,9 +4,14 @@ Public License (GPLv2+), its headers and libraries under version 2.1 or
 later of the less restrictive GNU Lesser General Public License (LGPLv2.1+),
 its documentation under version 4.0 or later of the Creative Commons
 Attribution-ShareAlike International Public License (CC-BY-SA v4.0+),
-and its init scripts under the Revised BSD license.
+and its init scripts under the Revised BSD license.  Some small, ancillary
+files (kept away from authoritative Pacemaker redistributables at this time)
+use the Free Software Foundation All Permissive (FSFAP) license, as noted
+in the files themselves.
```

Comment:
We can drop the "kept away" clause. The entire git repository is an authoritative redistributable; the various ways to consume subsets of it are simply a convenience.